libcxx/test/std/utilities/meta/meta.trans/visibility.sh.cpp

```diff
@@ -0,0 +1,71 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Meta-comment: I think I would group all these tests in a single file under `test/std/utilities/meta`, just to avoid duplicating the `RUN:` boilerplate.

Reply:
I mean, it's not that much boiler plate. I'd rather avoid having huge test files if they can be split somewhat meaningfully. We could also consider adding a `.shared.cpp` test or something like that if we want to extend our test coverage in this area in the future.

Reply:
Ack, I don't feel strongly enough to block the review on that.